Common/Trees/src/main/java/org/tribuo/common/tree/ExtraTreesTrainer.java

```diff
@@ -75,6 +81,22 @@ public ExtraTreesTrainer(DecisionTreeTrainer<T> trainer, EnsembleCombiner<T> com
         postConfig();
     }
 
+    /**
+     * Constructs an ExtraTreesTrainer with parallel training support.
+     * <p>
+     * Throws {@link PropertyException} if the trainer is not set to use random split points.
+     * @param trainer The tree trainer.
+     * @param combiner The combining function for the ensemble.
+     * @param numMembers The number of ensemble members to train.
+     * @param seed The RNG seed.
+     * @param numThreads The number of threads to use for parallel training.
+     *                   Use USE_ALL_AVAILABLE_PROCESSORS to use all available processors, 1 for single-threaded.
```

Comment:
This should be a link to the field `{@link BaggingTrainer#USE_ALL_AVAILABLE_PROCESSORS}`.

Reply:
Addressed in 3d5ba968a33002860fb05baa9a9d560731b804cb